src/bin/cargo/commands/vendor.rs

```diff
@@ -7,59 +7,57 @@ pub fn cli() -> App {
         .about("Vendor all dependencies for a project locally")
         .arg_quiet()
         .arg_manifest_path()
-        .arg(Arg::with_name("path").help("Where to vendor crates (`vendor` by default)"))
         .arg(
-            Arg::with_name("no-delete")
+            Arg::new("path")
+                .allow_invalid_utf8(true)
```

Comment:
> value_of_os now requires setting allow_invalid_utf8, otherwise it
asserts. To help catch this, I updated the argument definitions
associated with lookups reported by: 

I thought it interesting that most clap path arguments don't allow non-UTF-8 arguments but `vendor` seems to be one of the main exceptions

Reply:
That's interesting. In general, any path argument we accept should allow non-UTF-8, but that can get fixed in a follow-up PR.

Reply:
The follow up PR would be two parts
- Add `arg.allow_invalid_utf8(true)`
- Call `value_of_os` rather than `value_of`